tests/qos/test_qos_sai.py

```diff
@@ -711,29 +748,14 @@ def testQosSaiSharedReservationSize(
             Raises:
                 RunAnsibleModuleFail if ptf test fails
         """
-
         qosConfig = dutQosConfig["param"]
         src_dut_index = get_src_dst_asic_and_duts['src_dut_index']
         src_asic_index = get_src_dst_asic_and_duts['src_asic_index']
         dst_dut_index = get_src_dst_asic_and_duts['dst_dut_index']
         dst_asic_index = get_src_dst_asic_and_duts['dst_asic_index']
         src_testPortIps = dutConfig["testPortIps"][src_dut_index][src_asic_index]
         dst_testPortIps = dutConfig["testPortIps"][dst_dut_index][dst_asic_index]
-
-        if not sharedResSizeKey in qosConfig.keys():
-            pytest.skip("Shared reservation size parametrization '%s' is not enabled" % sharedResSizeKey)
-
-        if "skip" in qosConfig[sharedResSizeKey]:
-            # Skip if buffer pools and profiles are not be present, marked by qos param generator
-            pytest.skip(qosConfig[sharedResSizeKey]["skip"])
-
-        self.updateTestPortIdIp(dutConfig, get_src_dst_asic_and_duts, qosConfig[sharedResSizeKey])
-
-        src_port_idx_to_id = list(src_testPortIps.keys())
-        dst_port_idx_to_id = list(dst_testPortIps.keys())
-        # Translate requested port indices to available port IDs
-        src_port_ids = [src_port_idx_to_id[idx] for idx in qosConfig[sharedResSizeKey]["src_port_i"]]
-        dst_port_ids = [dst_port_idx_to_id[idx] for idx in qosConfig[sharedResSizeKey]["dst_port_i"]]
+        (src_port_ids, dst_portids) = check_port_count
```

Comment:
renaming dst_port_ids  to dst_portids  caused undefined symbol error:

```
        (_, src_port_ids, dst_portids) = check_skip_shared_res_test
    
        testParams = dict()
        testParams.update(dutTestParams["basicParams"])
        testParams.update({
            "testbed_type": dutTestParams["topo"],
            "dscps": qosConfig[sharedResSizeKey]["dscps"],
            "ecn": qosConfig[sharedResSizeKey]["ecn"],
            "pgs": qosConfig[sharedResSizeKey]["pgs"],
            "queues": qosConfig[sharedResSizeKey]["queues"],
            "src_port_ids": src_port_ids,
            "src_port_ips": [src_testPortIps[port]['peer_addr'] for port in src_port_ids],
>           "dst_port_ids": dst_port_ids,
            "dst_port_ips": [dst_testPortIps[port]['peer_addr'] for port in dst_port_ids],
            "pkt_counts":  qosConfig[sharedResSizeKey]["pkt_counts"],
            "shared_limit_bytes": qosConfig[sharedResSizeKey]["shared_limit_bytes"],
            "hwsku":dutTestParams['hwsku']
        })
E       NameError: global name 'dst_port_ids' is not defined
```

Try to fix it in PR https://github.com/sonic-net/sonic-mgmt/pull/8644

suggest to run test to avoid baisic syntax error.